crates/ruff/tests/snapshots/integration_test__rule_f401.snap

```diff
@@ -34,16 +34,32 @@ marking it as unused, as in:
 from module import member as member
 ```
 
+Within `__init__.py` re-export a symbol with an `__all__` declaration containing a string
+matching the symbol, as in:
+
+```python
+# __init__.py
+import some_module
+
+__all__ = [ "some_module"]
+```
+
 ## Fix safety
 
-When `ignore_init_module_imports` is disabled, fixes can remove for unused imports in `__init__` files.
-These fixes are considered unsafe because they can change the public interface.
+Fixes to remove unused imports are safe, except in `__init__.py` files, where they could
+change the public interface. Fixes in `__init__.py` files are currently gated within preview
+mode.
```

Comment:
I'd say something like 

> Applying fixes to `__init__.py` files is currently in preview.

Maybe you can just move this to the start of the next paragraph instead of saying "In preview mode..."?